### PR TITLE
Update CITATION.cff

### LINKED ISSUE #478: [Documentation]: `CITATION.cff` needs updating?
### What would you like changed or added to the documentation and why?

Hello,
I believe the title and URL of the [CITATION.cff](https://github.com/catalystneuro/neuroconv/blob/main/CITATION.cff) still reference the deprecated NWB Conversion Tools.
Happy to make a quick PR if that can help.

### Do you have any interest in helping write or edit the documentation?

Yes.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/catalystneuro/neuroconv/blob/main/.github/CODE_OF_CONDUCT.rst)
- [X] Have you ensured this bug was not already [reported](https://github.com/catalystneuro/neuroconv/issues)?

### PR DESCRIPTION
**Pull request by bendichter**
fix #478

### CODE REVIEW

**Comment by CodyCBakerPhD** [comment]
While we're updating it, should we alter any authors as well?

In particular @weiglszonja has contributed a lot and ought to be included in it IMO

**Comment by bendichter** [comment]
@CodyCBakerPhD good idea. Added.

**Comment by CodyCBakerPhD** [comment]
Cool - going to let @weiglszonja approve and merge this one to ensure their name is written out how they would like it to be in a citation

File: CITATION.cff [hunk 1]
```diff
@@ -4,20 +4,23 @@ authors:
   - family-names: Baker
     given-names: Cody
     orcid: https://orcid.org/0000-0002-0829-4790
-  - family-names: Sharda
-    given-names: Saksham
   - family-names: Mayorquin
     given-names: Heberto
     orcid: https://orcid.org/0000-0002-5937-7537
+  - family-names: Weigl
+    given-names: Anna
```
Comment (weiglszonja) [inline CITATION.cff:11]:
```suggestion
    given-names: Anna Szonja
```

**Review by weiglszonja — APPROVED** [review]
Thank you for including me as well. Updated the name, looks good ☺️

**Pull request merged**